frontend/src/modules/financing-resource/form.jsx

```diff
@@ -170,19 +170,12 @@ const AddResourceForm = withRouter(
     highlight,
     setHighlight,
     setDisabledBtn,
+    isLoaded,
     history,
     match: { params },
   }) => {
-    const {
-      countries,
-      organisations,
-      tags,
-      currencies,
-      loading,
-      formStep,
-      formEdit,
-    } = UIStore.currentState;
-
+    const globalState = UIStore.useState();
```

Comment:
https://lostpebble.github.io/pullstate/docs/use-store-state-hook#use-a-sub-state-of-a-store